src/imagings/requests/NewImagingRequest.tsx

```diff
@@ -46,10 +48,21 @@ const NewImagingRequest = () => {
   useAddBreadcrumbs(breadcrumbs)
 
   const onPatientChange = (patient: Patient) => {
-    setNewImagingRequest((previousNewImagingRequest) => ({
-      ...previousNewImagingRequest,
-      patient: patient.fullName as string,
-    }))
+    if (patient) {
+      setNewImagingRequest((previousNewImagingRequest) => ({
+        ...previousNewImagingRequest,
+        patient: patient.fullName as string,
+      }))
+
+      const visits = patient.visits?.map((v) => ({ label: v.type, value: v.id }))
```

Comment:
I think the label should be something like `${visitType} at `${startDateTime}` or `${visitType} (${startDateTime)}`